### ISSUE TITLE
SVG compound path issue (likely involving edge pixels)

### ISSUE THREAD

**Issue by monkeyArms** [jankovicsandras/imagetracerjs#11]
Hello, this is an amazing utility that has been working great for me until I found an issue with the following image (8-bit png w/transparency):

![test-1-advice-10x](https://user-images.githubusercontent.com/1403262/30935837-8d74a782-a397-11e7-93a2-5e8d3a32ccc2.png)

The resulting svg has a flaw:  it seems to have a compound path joining 2 areas that should not be connected, illustrated here:

![ss](https://user-images.githubusercontent.com/1403262/30937371-2603199e-a39c-11e7-8061-61c144eeebe3.jpg)

I have tried playing with every applicable setting, but no luck.

Code:

```
var settings = {
	ltres:				1,
	qtres:				1,
	pathomit:			8,
	rightangleenhance:	false,
	blurradius:			0
};

ImageTracer.imageToSVG(
	'test-1-advice-10x.png',
	function( svgStr ) { /* do stuff */ },
	settings
);

```

**Comment by monkeyArms** [comment]
I wanted to add that I found a workaround for this - I added a transparent 5px border to my source image, applied ImageTracer, then cropped the border back out of the resulting SVG by altering the width, height and viewBox properties.

**Comment by jankovicsandras** [comment]
Hi,

Thanks for reporting and for the workaround! :) I'm not 100% sure what's the cause, but I'll try to fix it.

**Comment by jankovicsandras** [comment]
I hope this is fixed now in the new 1.2.2 version. The problem was incorrect initial values for a bounding box, which is required to find the parent shape of a hole.

**Comment by monkeyArms** [comment]
Thank you so much.  This library is a lifesaver on a project I am working on!
